### PR TITLE
added calendar toggle styling, aligned section title, toggle, and actions to offerings

### PR DESCRIPTION
**Pull request by thecoolestguy**
Styled the header row to include inline/responsive title, toggle-switch, and action dropdown.  Added Calendar toggle placeholder, but there is currently no calendar component in-place.  The _toggles.scss needs consolidation

### CODE REVIEW

File: app/styles/components/_toggles.scss [hunk 1]
```diff
@@ -212,14 +212,43 @@ switch-wide > .switch-input:checked ~ .switch-handle {
...
-  left: 104px;
+  left: 96px;
   box-shadow: -1px 1px 5px rgba(0, 0, 0, 0.2);
 }
 
+.switch.calendar {
+  width: 120px;
+
+  .switch-input:checked ~ .switch-label {
+    background: #5accef;
```
Comment (jrjohnson) [inline app/styles/components/_toggles.scss]: Should be a variable name not a hardcoded color

File: app/styles/components/_toggles.scss [hunk 2]
```diff
@@ -212,14 +212,43 @@ switch-wide > .switch-input:checked ~ .switch-handle {
...
+  }
+
+  .switch-input ~ .switch-label {
+    background: $toggle-background-grey;
+    color: #ddd;
```
Comment (jrjohnson) [inline app/styles/components/_toggles.scss]: Should be a variable name not a hardcoded color

**Comment by jrjohnson** [comment]
Code looks good, I didn't star it because we will probably have to rebase to get tests to pass.  @saschaben please review for aesthetics.

**Comment by saschaben** [comment]
appears to be fine. noticed a possibly related issue --- when opening the "add new" offering form (I know it's still unstyled), and hitting the cancel button, we get the following detritus left on screen:

ilios 2015-04-23 20-02-25

If related, please fix. Otherwise, let's address where necessary.

**Comment by thecoolestguy** [comment]
I just noticed the actual header part... Closing for now.  Will fix shortly.

**Pull request closed without merging**